services/wallet/service.go

```diff
@@ -103,6 +105,11 @@ type GeoValidator interface {
 	Validate(ctx context.Context, geolocation string) (bool, error)
 }
 
+type Metric interface {
```

Comment:
Change request: Does this interface have to be exported? I don't think so. Let's make it private, if the situation allows.